docs/tsm/TSM_FORMAT.md

```diff
@@ -0,0 +1,161 @@
+# TSM Inverted Index Proposal
+
+This is a proposal for adding a disk-based, inverted index for measurements, tags and values.  This is a extension and/or alternative of the ideas proposed in [TSI Proposal #7174](https://github.com/influxdata/influxdb/pull/7174).
+
+The inverted index supports the following:
+
+* Hash lookups of measurement, tags, and values
+* Sorted iteration over measurement, tags and values
+* Efficient series boolean operations over compressed bitmaps
+* Compressed series keys
+
+The index is a separate set of files specific to a shard that are compacted and merged periodically.  Searches run across multiple files and are merged.
+
+## Format
+
+┌────────────────────────────────────────────────────────────────┐
```

Comment:
Can you put the diagrams in triple back-ticks so they show up properly on the markdown document?